src/harness/sas_interface.py

```diff
@@ -374,6 +374,21 @@ def TriggerPpaCreation(self, request):
     """
     pass
 
+  def TriggerDailyActivitiesImmediately(self):
+    """SAS admin interface to trigger daily activities immediately which will
+    execute the activities mentioned in the SAS admin agreement
```

Comment:
It's probably best to spell them out explicitly so that (1) the information is available to everyone and (2) it's very clear what's going on here (since we actually differ from the doc that Masoud et al. are working on with SAS admins).